src/scripts/validation.js

```diff
@@ -0,0 +1,99 @@
+export const validationConfig = {
+  formSelector: ".modal__form",
+  inputSelector: ".modal__input",
+  submitButtonSelector: ".modal__submit-btn",
+  inactiveButtonClass: "modal__submit-btn-disabled",
```

Comment:
`modal__submit-btn-disabled`

It should be a **modifier** of `modal__submit-btn`, so it should be `modal__submit-btn_disabled`, according to BEM